### DISCUSSION TITLE
[Question] what source code version does rke2 use to build rancher/hardened-cluster-autoscaler:v1.8.6-build20230609

### DISCUSSION THREAD

**Discussion by went2** [rancher/rke2#6041] [Q&A] [answered]
Hi, I'm working on replacing some RKE2 built-in images with ones  built from source code.

I ran into a problem while trying to create an image of rancher/hardened-cluster-autoscaler:**v1.8.6-build20230406**. The [source code repository](https://github.com/rancher/image-build-cluster-proportional-autoscaler) starts at tag v1.8.10 and doesn't have a tag for v1.8.6.

Could you let me know where I can find the source code for the v1.8.6-build20230406 version of the autoscaler? Any help would be really appreciated. Thanks!

**Comment by manuelbuil** [comment, marked as answer]
Let me edit my answer. We were building v1.8.6 out of the coredns github repo but we decided to create its own github repo a few months ago as it will make it easier to maintain. This is the PR doing that: https://github.com/rancher/image-build-coredns/pull/36

**Reply by went2** [reply]
Thanks for your explanation, it's helpful!
